buildscripts/kokoro/android.sh

```diff
@@ -91,6 +91,7 @@ new_apk_size="$(stat --printf=%s $HELLO_WORLD_OUTPUT_DIR/apk/release/app-release
 cd $BASE_DIR/github/grpc-java
 git checkout HEAD^
 ./gradlew clean
+./gradlew --stop  # use a new daemon to build the previous commit
```

Comment:
This should go above the `./gradlew clean` since that clean is running at the new commit, or the `./gradlew clean` should be moved before the `git checkout HEAD^` (preferable).

Reply:
Fixed.